### ISSUE TITLE
Use scf v2 variables names to allow for easier v3 upgrade experience.

### ISSUE THREAD

**Issue by gaktive** [SUSE/scf#2798]

**Comment by drnic** [comment]
For my education - what does "V1" and "V2" mean in context here? And how does it related to SCF v3?

**Comment by jandubois** [comment]
I think this is a mixup, @gaktive is probably referring to CAP-1.x and CAP-2.x, where CAP-1.x is based on SCFv2, and CAP-2.x will be based on SCFv3.

CAP being the SUSE "Cloud Application Platform".

**Comment by gaktive** [comment]
Yep, that was internal versioning mapped badly to SCF as I was in copy pasta mode. All apologies.

**Comment by gaktive** [comment]
Reworded this story as the importance of the upgrade experience for CAP 1 (or scf v2) users needs to be highlighted. The current discussion about converting YAML values to be v3 appropriate seems daunting. We need a spike to see what we can maintain and convert here.

**Comment by gaktive** [comment]
To be worked on once we get more functionality into v3 for feature parity.
